--- soma.SlackBuild.orig	2017-12-02 11:26:26.000000000 -0600
+++ soma.SlackBuild	2017-12-02 11:48:00.554981067 -0600
@@ -26,7 +26,7 @@
 VERSION=${VERSION:-2.12.3}
 ARCH=noarch
 BUILD=${BUILD:-1}
-TAG=${TAG:-daw}
+#TAG=${TAG:-daw}
 
 CWD=$(pwd)
 TMP=${TMP:-/tmp/daw}
@@ -51,16 +51,16 @@
 mkdir -p $PKG/usr/bin $PKG/etc/soma/themes $PKG/install $PKG/usr/doc/$PRGNAM-$VERSION
 
 install -m 755 soma $PKG/usr/bin/soma
-install -m 644 options.conf $PKG/etc/soma
-install -m 644 stations.conf $PKG/etc/soma
+install -m 644 options.conf $PKG/etc/soma/options.conf.new
+install -m 644 stations.conf $PKG/etc/soma/stations.conf.new
 install -m 644 themes/* $PKG/etc/soma/themes
 
 cp -a \
   README $CWD/ChangeLog.txt \
   $PKG/usr/doc/$PRGNAM-$VERSION
 
-cat $CWD/$PRGNAM.SlackBuild > $PKG/usr/doc/$PRGNAM-$VERSION/$PRGNAM.SlackBuild
 cat $CWD/slack-desc > $PKG/install/slack-desc
+zcat $CWD/doinst.sh.gz > $PKG/install/doinst.sh
 
 cd $PKG
-/sbin/makepkg -l y -c n $OUTPUT/$PRGNAM-$VERSION-$ARCH-$BUILD$TAG.${PKGTYPE:-tgz}
+/sbin/makepkg -l y -c n $OUTPUT/$PRGNAM-$VERSION-$ARCH-$BUILD$TAG.${PKGTYPE:-txz}
