--- rescan-scsi-bus.sh.orig	2012-01-14 16:43:33.000000000 -0600
+++ rescan-scsi-bus.sh	2012-09-07 12:51:52.003994404 -0500
@@ -625,11 +625,11 @@
     sg_version=$(sg_inq -V 2>&1 | cut -d " " -f 3)
     sg_version=${sg_version##0.}
     #echo "\"$sg_version\""
-    if [ -z "$sg_version" -o "$sg_version" -lt 70 ] ; then
-        sg_len_arg="-36"
-    else
+    #if [ -z "$sg_version" -o "$sg_version" -lt 70 ] ; then
+        #sg_len_arg="-36"
+    #else
         sg_len_arg="--len=36"
-    fi
+    #fi
 else
     echo "WARN: /usr/bin/sg_inq not present -- please install sg3_utils"
     echo " or rescan-scsi-bus.sh might not fully work."     
